src/encoded/tests/test_schema_experiment.py

```diff
@@ -123,3 +130,10 @@ def test_experiment_submission_date_dependency(testapp, experiment_no_error):
         'date_submitted': '2000-10-10'},
         status=200)
     
+def test_experiment_possible_controls(testapp, experiment_no_error, matched_set):
```

Comment:
Looks good. I would put this all on one line `{'possible_controls': [matched_set_control['@id']]}` and perhaps change expt -> exp. 

`def test_experiment_possible_controls(testapp, experiment_no_error, matched_set):
    exp = testapp.post_json('/experiment', experiment_no_error).json['@graph'][0]
    matched_set_control = testapp.post_json('/matched_set', matched_set).json['@graph'][0]
    testapp.patch_json(
        exp['@id'],
        {'possible_controls': [matched_set_control['@id']]},
        status=200
    )`